diff --git a/CMakeLists.txt b/CMakeLists.txt
index 3d14f50..f67601f 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -432,7 +432,7 @@ set (WXWIDGETS_FORCE_VERSION CACHE VERSION "Force usage of a specific wxWidgets
 if(WXWIDGETS_FORCE_VERSION)
   set (wxWidgets_CONFIG_OPTIONS --version=${WXWIDGETS_FORCE_VERSION})
 endif()
-FIND_PACKAGE(wxWidgets REQUIRED)
+FIND_PACKAGE(wxWidgets COMPONENTS ${wxWidgets_USE_LIBS} REQUIRED)
 
 IF(MSYS)
 # this is to convert msys to windows paths, and handle the missing /usr
